src/@rocketseat/gatsby-theme-docs/components/Docs/EditGithub.js

```diff
@@ -1,9 +1,9 @@
 /* @jsx jsx */
 import React from 'react'
-import { jsx, css } from '@emotion/core'
+import { jsx, css } from '@emotion/react'
```

Comment:
whats the purpose of jsx

Reply:
The use of `jsx` was not necessary here. This has been removed.